example/Index.re

```diff
@@ -237,7 +237,16 @@ App.get app path::"/xhr" @@ Middleware.from (fun req res next => {
   };
 });
 
-App.listen app port::3000 ;
+let onListen port e =>
+  switch e {
+  | exception (Js.Exn.Error e) =>
+    Js.log e;
+    Node.Process.exit 1
+  | _ => Js.log @@ "Listening at http://127.0.0.1:" ^ (string_of_int port) 
+  };
+
+let listening = onListen 3000;
+App.listen app port::3000 onListen::listening
```

Comment:
sorry for the runaround, this should change to match the updates outlined in the express.re file.